azure-monitor-opentelemetry/azure/monitor/opentelemetry/util/configurations.py

```diff
@@ -4,9 +4,56 @@
 # license information.
 # --------------------------------------------------------------------------
 
+from json import loads
+from logging import NOTSET
+from os import environ
 from typing import Dict
 
+from azure.monitor.opentelemetry._constants import (
+    DISABLE_LOGGING_ARG,
+    DISABLE_METRICS_ARG,
+    DISABLE_TRACING_ARG,
+    EXCLUDE_INSTRUMENTATIONS_ARG,
+    INSTRUMENTATION_CONFIG_ARG,
+    LOGGER_NAME_ARG,
+    LOGGING_EXPORT_INTERVAL_MS_ARG,
+    LOGGING_LEVEL_ARG,
+    METRIC_READERS_ARG,
+    SAMPLING_RATIO_ARG,
+    TRACING_EXPORT_INTERVAL_MS_ARG,
+    VIEWS_ARG,
+)
 from azure.monitor.opentelemetry._types import ConfigurationValue
+from opentelemetry.sdk.environment_variables import (
+    OTEL_LOG_LEVEL,
+    OTEL_TRACES_SAMPLER_ARG,
+)
+
+
+_CONFIGURATION_ENV_VAR_PREFIX = "APPLICATIONINSIGHTS_"
```

Comment:
This was chosen to match connection string. However, other good options are AZURE_MONITOR_ ,AZURE_MONITOR_OTEL_, AZ_MON_ ...etc